chad/cards-template1a.js

```diff
@@ -0,0 +1,63 @@
+// Simple version (no error-detection)
```

Comment:
Looks good. Consider ways you can reduce some of these functions even further. For example, in your rank function, you don't necessarily need to both create a var rankIndex equal to an equation and then return it; instead you can simply return the equation without a variable. Nice work overall here.